gr-digital/python/digital/qa_ofdm_frame_equalizer_vcvc.py

```diff
@@ -167,6 +167,8 @@ def test_001c_carrier_offset_cp (self):
         # Check data
         self.assertComplexTuplesAlmostEqual(rx_expected, sink.data()[0], places=4)
 
+
+    @gr_unittest.unittest.skip("FIXME: test fails")
```

Comment:
what's up wtih this? or is the test just broken in general?

Reply:
Since this is the non-sliced output, it's hard to test. How do we usually test the analog output of a complicated block?

Reply:
@noc0lour It was already broken, at least on my machine.

Reply:
This test works without the patch. Something needs updating ... hunting.

Reply:
I get this error using current master: https://gist.github.com/japm48/0d01120518c4311138284c421a927194
I'll try to dig in and see what's happening.

Reply:
I know what that is. I see it on ARM.  It's caused by the compiler flag ```-fcx-limited-range```.

https://github.com/gnuradio/gnuradio/pull/3114

Reply:
I think this is just a path issue. The test fails for me if there is an existing install in my path ... it picks up the wrong gr-digital libraries. Give it a try with no env set. Unfortunately that means making a parallel installation of Volk somewhere. First, just try enabling the test and see if it runs under buildbot.

Reply:
Ye, the test python stuff is messed up w.r.t to existing libraries and for the tests to run successfully you have to install GNU Radio because the python library is only assembled then. (IIRC)

Reply:
> I get this error using current master: https://gist.github.com/japm48/0d01120518c4311138284c421a927194
> I'll try to dig in and see what's happening.

Shouldn't be almostEqual and using the last (5) digits be enough precision for that?

Reply:
@drmpeg Thanks! I can confirm that `-fcx-limited-range` was the culprit (I recompiled after removing the flag). Volk is unrelated here (I'm using the system package in Arch). In case it's needed: gcc version 10.2.0.

@willcode No duplicate library problem here, I already learned the hard way that I need to avoid that.

@noc0lour I'm guessing it wasn't done that because assertAlmostEqual doesn't work on dicts, so I'll use a simple workaround.